tests/integration_test/shadow_system_test.c

```diff
@@ -817,11 +817,17 @@ TEST_SETUP( deviceShadow_Integration )
 
 TEST_TEAR_DOWN( deviceShadow_Integration )
 {
+    MQTTStatus_t status;
+
     /* Terminate MQTT connection. */
-    TEST_ASSERT_EQUAL( MQTTSuccess, MQTT_Disconnect( &context ) );
+    status = MQTT_Disconnect( &context );
 
     /* Terminate TLS session and TCP connection. */
     ( void ) SecureSocketsTransport_Disconnect( &networkContext );
+
```

Comment:
Same comment as previous file – why are we discarding the return value on line 826?  It seems like we should be validating it just as we validate the return value of `MQTT_Disconnect()`.